tests/test_dagmc_query.py

```diff
@@ -219,3 +200,77 @@ def test_duplicate_calc(function, message):
             if message in str(w[-1].message):
                 test_pass[1] = True
     assert(all(test_pass))
+
+def test_coarseness():
+    """Tests the calc coarseness function
+    """
+    three_vols = df.DagmcFile(test_env['three_vols'])
+    vol = three_vols.entityset_ranges['volumes'][0]
+    three_vols_query = dq.DagmcQuery(three_vols, vol)
+    three_vols_query.calc_coarseness()
+    np.testing.assert_almost_equal(
+        list(three_vols_query._surf_data['coarseness']), list(np.full(6, 0.02)))
+
+def test_coarseness_area_called():
```

Comment:
I am not sure this test is necessary. It isn't testing the functionality of the coarseness function but rather the appropriate behavior of the triangle area function, which is already tested for.